frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx

```diff
@@ -155,6 +167,62 @@ export function FunnelTab(): JSX.Element {
                                 useMultiBreakdown={!!featureFlags[FEATURE_FLAGS.BREAKDOWN_BY_MULTIPLE_PROPERTIES]}
                             />
                         </Row>
+                        {featureFlags[FEATURE_FLAGS.BREAKDOWN_ATTRIBUTION] && (
+                            <>
+                                <h4 className="secondary mt">
+                                    Attribution
+                                    <Tooltip placement="right" title="filler">
+                                        <InfoCircleOutlined className="info-indicator" />
+                                    </Tooltip>
+                                </h4>
+                                <Row>
+                                    <LemonSelect
+                                        placeholder="Attribution"
+                                        options={{
+                                            [BreakdownAttributionType.FirstTouch]: { label: 'First touchpoint' },
+                                            [BreakdownAttributionType.LastTouch]: { label: 'Last touchpoint' },
+                                            ...(filters.funnel_order_type === StepOrderValue.UNORDERED
+                                                ? { [BreakdownAttributionType.AnyStep]: { label: 'Any step' } }
+                                                : {
+                                                      [BreakdownAttributionType.Step]: {
+                                                          label: 'Specific step',
+                                                          element: (
+                                                              <LemonSelect
+                                                                  outlined
+                                                                  className="ml-05"
+                                                                  onChange={(value) => {
+                                                                      if (value) {
+                                                                          setFilters({
+                                                                              breakdown_attribution_type:
+                                                                                  BreakdownAttributionType.Step,
+                                                                              breakdown_attribution_value: value,
+                                                                          })
+                                                                      }
+                                                                  }}
+                                                                  placeholder={`Step ${
+                                                                      filters.breakdown_attribution_value || 1
```

Comment:
quick flyby that `breakdown_attribution_value` is zero indexed while steps are 1 indexed 😅

Reply:
:P

Reply:
wait is value also 1 indexed?

Reply:
what do you mean? 😅 .

`breakdown_attribution_value` follows the 'order' in the entity definition. If order = 0, if you want step 1, value = 0 too.

If you want step = 3, value = 2, order = 2